index.js

```diff
@@ -0,0 +1,210 @@
+const inquirer = require("inquirer");
+const fs = require("fs");
+
+// All Licenses Obj, with categories. Source: https://gist.github.com/lukas-h/2a5d00690736b4c3a7ba (better be imported via an API)
```

Comment:
remove comment if no longer needed or add more info so that the user can better understand please